app/models/book.rb

```diff
@@ -0,0 +1,32 @@
+class Book < ApplicationRecord
+  include ActionView::Helpers::NumberHelper
+
+  validates :name, :price, :description, presence: true
+  validates :height, :width, :depth, :price, :year, numericality: { greater_than_or_equal_to: 0.01 }
+  has_and_belongs_to_many :authors
+  belongs_to :category
+  has_many :positions
+  has_many :reviews
+
+  mount_uploaders :images, ImageUploader
+
+  def short_description
+    description.truncate(100, separator: ' ')
```

Comment:
usually, it is a good idea to move such numbers to a constant.
Add a constant `SHORT_DESCRIPTION_LENGHT = 100` and use it here instead of a `magic number`